service/component-samples/reseller/src/main/java/org/fido/iot/sample/ResellerConfigLoader.java

```diff
@@ -38,6 +38,7 @@ public static String loadConfig(String property) {
       try {
         fileBasedConfiguration = builder.getConfiguration();
       } catch (ConfigurationException e) {
+        System.out.println("Application might not be using config file");
```

Comment:
Should this be using the logger instead?

Reply:
We will move to using the logger in a subsequence PR.